packages/audio/fdk-aac/package.mk

```diff
@@ -0,0 +1,20 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (C) 2022-present Marek Moeckel (wansti@discarded-ideas.org)
+
+PKG_NAME="fdk-aac"
+PKG_VERSION="2.0.2"
+PKG_SHA256="7812b4f0cf66acda0d0fe4302545339517e702af7674dd04e5fe22a5ade16a90"
+PKG_LICENSE="Apache2"
```

Comment:
I'm concerned about the license on this, as it's not open source compatible.

https://github.com/mstorsjo/fdk-aac/blob/master/NOTICE

https://www.gnu.org/licenses/license-list.en.html#fdk

Reply:
Updated as per our discussion on Discord.